SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+The OPI security policy aims to efficiently handle security vulnerabilities
+while minimizing public disclosure until a fix is available.
+
+Reporters are requested to submit security issues privately via the
+[GitHub advisory](https://github.com/opiproject/opi/security/advisories/new)
+system.
+
+Reports should be acknowledged within 5 days. OPI maintainers will try to
+respond to a report within a maximum of 10 days, indicating the next steps
+in handling the submission (although it will depend on the severity of the
+report).
+
+## Reporting a Vulnerability in 3rd Party Modules
+
+Security issues in third-party modules should be reported according to their
+own security policies or directly to their respective maintainers if no such
+security policy is in place for the respective module.
+
+## Vulnerability Disclosure Policy
+
+Upon receiving a security report, OPI follows the Common Vulnerabilities and
+Exposures (CVE®) process outlined below:
+
+1. OPI maintainer acknowledges the receipt of the report and calls for a
+maintainer/security experts meeting to discuss the scope of the vulnerability
+and identify any potentially related problems.
+
+2. Resources are identified and assigned to investigate the issue in more detail
+so that a fix or workaround can be proposed.
+
+3. The reporter is notified about the next steps in handling the vulnerability
+and may be invited to collaborate on a fix.
+
+4. Once a fix or workaround is prepared, it is submitted for a standard review
+by creating a pull request (PR) *without disclosing any details about the
+vulnerability* but merely stating what the patch does.
+
+5. Once the PR(s) is merged, the most recent official release will be used as
+a baseline for a maintenance release and will include only the fix(es)
+identified for the issue at hand.
```

Comment:
At some point in the process, perhaps here, the CVSS score for the issue should be determined so that the initial value can be included in the CVE.  The score can be calculated using the CVSS calculator (https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator)

Reply:
We can do that of course but I have not seen this explicitly mentioned in other disclosure processes - just fill in CVE as a step. Do you think we should mention this as a separate step or rather treat it as part of internal handling?

Reply:
I decided to add this after Joe's comment and linked to CVSS calc to have a unified approach for the severity computation